install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root."
+   exit 1
+fi
+
+PREFIX=/usr
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+# Install JobAdder source code + binaries
+# FIXME: how to make setup.py ignore test folder??
+cd $SCRIPTPATH/src/
+mv ./test/__init__.py ./test/init
+./setup.py install --prefix=$PREFIX
```

Comment:
Shouldn't we be using pip to install our project?
I think setup.py install doesn't support automatic uninstallation.

Reply:
I changed to pip, thanks for the suggestion.